doc/rst/meta/modules/packages.rst

```diff
@@ -6,14 +6,116 @@ Package Modules
 Package modules are libraries that currently live outside of the
 Chapel Standard Library, either because they are not considered to be
 fundamental enough or because they are not yet mature enough for
-inclusion there.
+inclusion there.  Over time, we expect many of these to become
+``mason`` packages.
 
+Algorithms
+----------
+.. toctree::
+   :maxdepth: 1
+
+   Crypto <packages/Crypto>
+   Search <packages/Search>
+   Sort <packages/Sort>
+
+Communication (Inter-Locale)
+----------------------------
+.. toctree::
+   :maxdepth: 1
+
+   AllLocalesBarriers <packages/AllLocalesBarriers>
+   CopyAggregation <packages/CopyAggregation>
+   MPI <packages/MPI>
+   UnorderedAtomics <packages/UnorderedAtomics>
+   UnorderedCopy <packages/UnorderedCopy>
+
+Data Structures
+---------------
+.. toctree::
+   :maxdepth: 1
+
+   LinkedLists <packages/LinkedLists>
```

Comment:
Is this supposed to be in sorted order?

Reply:
[pushes quick commit]  "What?  Where are you quoting that from, I don't see it!?!"

Thanks Andrew, it's clearly back to first grade for me...